libs/utils/utils.js

```diff
@@ -257,63 +257,57 @@ export function loadStyle(href, callback) {
   return loadLink(href, { rel: 'stylesheet', callback });
 }
 
-export function appendHtmlPostfix(area = document) {
-  const pageUrl = new URL(window.location.href);
-  if (!pageUrl.pathname.endsWith('.html')) return;
+const pageUrl = new URL(window.location.href);
+const pathEndsWithHtml = pageUrl.pathname.endsWith('.html');
```

Comment:
Move to top of file if we're going to set these as "globals"

After that `pageUrl` can also be used on lines 116 & 646

Reply:
Thanks for the feedback! Implemented the suggestions in a new commit.